# Navigation goal coordinates
float64 latitude        # Target latitude in degrees (WGS84)
float64 longitude       # Target longitude in degrees (WGS84)
